libmscore/chord.cpp

```diff
@@ -247,18 +250,38 @@ Chord::Chord(const Chord& c, bool link)
       _arpeggio      = 0;
       _stemSlash     = 0;
       _tremolo       = 0;
-
       _graceIndex     = c._graceIndex;
       _noStem         = c._noStem;
       _playEventType  = c._playEventType;
       _stemDirection  = c._stemDirection;
       _noteType       = c._noteType;
       _crossMeasure   = CrossMeasure::UNKNOWN;
 
+      for (Note* onote : c._notes) {
+            Note* nnote;
+            if (fac)
+                  nnote = fac->cloneNote(onote, link);
+            else
+                  nnote = new Note(*onote, link);
+            add(nnote);
```

Comment:
That is also unusual because that JianpuChord class inherits from regular Chord anyway, so that already accounts for one copy of each note of the chord.  But then this code adds the notes again, which seems redundant.